rpc/src/v1/helpers/errors.rs

```diff
@@ -173,6 +174,14 @@ pub fn no_author() -> Error {
 	}
 }
 
+pub fn no_work_required() -> Error {
+	Error {
+		code: ErrorCode::ServerError(codes::NO_WORK_REQUIRED),
+		message: "External work is only required for Ethash engines.".into(),
```

Comment:
Yeah, but the comment says that it's required only for `Ethash engines`. In principle we could support other engines that work the same way as Ethash (Request work package -> Submit seal).
So my point was that the comment can be made more general.